modules/core/include/opencv2/core/utils/filesystem.hpp

```diff
@@ -0,0 +1,38 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_UTILS_FILESYSTEM_HPP
+#define OPENCV_UTILS_FILESYSTEM_HPP
+
+namespace cv { namespace utils { namespace fs {
+
+
+CV_EXPORTS bool isExist(const cv::String& path);
```

Comment:
it's wrong grammar. I suggest `doesExist()`, `isValidPath()`, `pathExist()`, `isFileOrDirectory()` etc.

Reply:
Changed to `exists()`: http://en.cppreference.com/w/cpp/filesystem/exists